crates/astria-bridge-contracts/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Add `contract_is_erc20` setting for watcher [#1762](https://github.com/astriaorg/astria/pull/1762)
```

Comment:
```suggestion
### Added

- Add `contract_is_erc20` setting for watcher [#1762](https://github.com/astriaorg/astria/pull/1762).
```

Reply:
Removed that setting. I guess the entry in `CHANGES` is now more appropriate (because I only change behavior but not the public API of the crate)?

Reply:
Yes, that makes sense to me.